static/css/style.css

```diff
@@ -6,6 +6,10 @@
   box-sizing: border-box;
 }
 
+html {
+  background-color: #fff;
```

Comment:
Would that work on the `body` element too? if it does, please move it to there.

Reply:
Yes that should work too, fixed.